localization/english/event_localization/wc_scourge_invasion_l_english.yml

```
@@ -4,4 +4,16 @@
  WCCSC_5_OPT_A_death:0 "All are equal in death."
  WCCSC_5_OPT_A_legion:0 "This is your last chance, [nerzhul.GetFirstName]. Serve, or be destroyed."
  WCCSC_5_OPT_A_evil:0 "Death has come into play at last..."
- WCCSC_5_OPT_A_neutral:0 "[ROOT.Char.GetFaith.HighGodName|U] save us all!"
\ No newline at end of file
+ WCCSC_5_OPT_A_neutral:0 "[ROOT.Char.GetFaith.HighGodName|U] save us all!"
+ 
+ WCCSC_15_TITLE:0 "Jailors of the Damned"
+ WCCSC_15_DESC:0 ""Did you think of yourself shrewd enough to get rid of us, orc?" [scourge_jailor.GetFirstName] asked. A hint of snark in [scourge_jailor.GetHerHis] voice as [scourge_jailor.GetHerHis] fiery eyes laid on you. Angered, powerless is how you felt.\n\nYou'd somehow manage to get rid of one of [scourge_jailor.GetHerHis] brothers, perhaps in a foolish bid to escape the gilded cage in which the Legion trapped you. Perhaps you should've stuck with doing what you were told to do, but there you were: powerless. As the demonic being tore the Frozen Throne apart, you pondered what could have been if you had succeeded."
+ RULKAN_OPT:0 "Rulkan, my love, I'm coming..."
+ 
+ WCCSC_20_DESC:0 "Perched atop Icecrown, where even the Light dared not show itself, you stared at the great snowy land of Northrend, pondering as your undying hordes of minions carried out your command ever so dutifully.\n\nSuddenly, a soft whisper was carried out by the cold wind. A dreadlord with a smug smirk on [scourge_jailor.GetHerHis] face stood there in front of your undead being, [scourge_jailor.GetHerHis] fiery eyes piercing through your icy cage: "There remains little time, [ROOT.GetCharacter.GetFirstName]. Five measly years is all you've left to fulfill your role." [scourge_jailor.GetSheHe] said, seemingly enjoying the thought of what [scourge_jailor.GetSheHe]'d do were you ever to live up to the Legion's expectations."
```

I feel like his amusement is more appropriate if he is thinking of you failing. Works as is, though, so this is just a suggestion.
```suggestion
 WCCSC_20_DESC:0 "Perched atop Icecrown, where even the Light dared not show itself, you stared at the great snowy land of Northrend, pondering as your undying hordes of minions carried out your command ever so dutifully.\n\nSuddenly, a soft whisper was carried out by the cold wind. A dreadlord with a smug smirk on [scourge_jailor.GetHerHis] face stood there in front of your undead being, [scourge_jailor.GetHerHis] fiery eyes piercing through your icy cage: "There remains little time, [ROOT.GetCharacter.GetFirstName]. Five measly years is all you've left to fulfill your role." [scourge_jailor.GetSheHe] said, seemingly enjoying the thought of what [scourge_jailor.GetSheHe]'d do were you ever to fail to live up to the Legion's expectations."
```